### PR TITLE
New script: Plot dipy sphere

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Was this available anywhere else? I could not find it. Just plotting the sphere.

Very easy to test: `scil_visualize_dipy_sphere.py symmetric323`

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Maybe I would have extended this script: scil_visualize_gradients.py
instead of creating a new one.

**Comment by EmmaRenauld** [comment]
> Maybe I would have extended this script: scil_visualize_gradients.py instead of creating a new one.

Ok, I can do that.

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by EmmaRenauld** [comment]
You can test with `scil_visualize_gradients.py --dipy_sphere 'repulsion724'`

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_visualize_gradients.py [hunk 1]
```diff
@@ -21,16 +23,23 @@
...
-    p.add_argument(
-        'in_gradient_scheme', nargs='+',
+    g = p.add_mutually_exclusive_group(required=True)
+    g.add_argument(
+        '--in_gradient_scheme', nargs='+',
         help='Gradient sampling filename. (only accepts .bvec and .bval '
              'together or only .b).')
+    g.add_argument(
+        '--dipy_sphere', choices=sphere_choices,
+        help="Sphere to show, amongst:\n{}".format(sphere_choices))
```
Comment (arnaudbore) [inline scripts/scil_visualize_gradients.py:42]: don't need to add amongst because choices are already printed in the help. Apart from that good to go !

**Pull request merged**